xapian-bindings/configure.ac

```diff
@@ -131,6 +149,10 @@ dnl CXXFLAGS for the duration of configure.
 save_CXXFLAGS=$CXXFLAGS
 CXXFLAGS="$CXXFLAGS $XAPIAN_CXXFLAGS"
 
+if test yes = "$enable_letor"; then
+    CXXFLAGS+="$XAPIANLETOR_CXXFLAGS"
```

Comment:
`+=` isn't portable (use the approach taken just above to append instead).  Also this is missing a space at the start of the appended value, so could result in CXXFLAGS having a value like `-I/opt/include/xapian-I/opt/include/xapian-letor`